examples/bri-3/prisma/seed.ts

```diff
@@ -19,7 +20,20 @@ async function main() {
       description: 'External Bpi Subject',
     },
   });
-  // TODO: Add Bpi Subjects needed for messaging as well as any other entities needed to have a local instance up and running
+
+  await prisma.bpiSubject.create({
+    data: {
+        name: 'BpiAdmin',
+        description: 'Internal Bpi Subject of this Bpi',
+        publicKey: '0x08872e27BC5d78F1FC4590803369492868A1FCCb', // private key 2c95d82bcd8851bd3a813c50afafb025228bf8d237e8fd37ba4adba3a7596d58
```

Comment:
let's not add private keys in code like this, no matter if it's some dummy private key, it should be env var

Reply:
It is really seed data, meant for testing purposes. I can add a big disclaimer saying that this is a private key used exclusively for testing. Would like to avoid additional complexity of needing secrets to run automated unit and\or e2e tests.